packages/grafana-ui/src/components/Logs/LogRowContextProvider.tsx

```diff
@@ -58,69 +60,74 @@ export const getRowContexts = async (
 
   const results: Array<DataQueryResponse | DataQueryError> = await Promise.all(promises.map(p => p.catch(e => e)));
 
-  return {
-    data: results.map(result => {
-      const dataResult: DataQueryResponse = result as DataQueryResponse;
-      if (!dataResult.data) {
-        return [];
-      }
+  const data = results.map(result => {
```

Comment:
Saving `data` and `errors` as variables instead of just returning them, so we can modify their order based on `logs sort order`